chrisj117/CS-472-GROUP-2-PROJECT
# Integrate review model with frontend

**WHAT DOES THIS PR DO?**

- Incorporates the backend API to the Review page.
- Ensures each course has a dynamically created review page, allowing users to view and submit reviews specific to each course.

**WHAT ISSUES ARE RELATED TO THIS PR?**

- #160

**HOW DO I TEST OUT THIS PR?**

- Navigate to the `web` directory and run `npm install` to install the necessary dependencies followed by `npm run dev` to start the development server.
- Navigate to the  `backend` directory in another terminal and make sure the backend server is running by executing `python3 manage.py runserver `.
- On your browser, go to the courses section and select any course. You should be directed to a dedicated review page for that course.
- The review page should display existing reviews (if any) and provide an option to write a new review.
- Submit a new review and verify if it is posted correctly and appears in the list of reviews.
- Ensure that the review page updates dynamically as you switch between different courses.
